### ISSUE TITLE
Reviewing Orgnames in the authority list

### ISSUE THREAD

**Issue by alliyya** [cwrc/ontology#426]
Revising ones that clearly shouldn't be organizations ex. Evangelicalism.
Revising standard names that are acronyms when they should be full forms, possible duplicates that have arised from this. ex. ANC and African National Congress.
Of course revising affected entries that contain the affected terms

**Comment by SusanBrown** [comment]
Now many instances are there of orgnames with such tags in the data? i.e. how many docs would need changing?


On Aug 7, 2018, at 12:20 AM, Alliyya Mo <notifications@github.com<mailto:notifications@github.com>> wrote:


Fair bit of ism's that probably shouldn't be in the authority list for organizations, some examples

        <AUTHORITYITEM STANDARD="Buddhism"/>
        <AUTHORITYITEM STANDARD="Communism"/>
        <AUTHORITYITEM STANDARD="Deism"/>
        <AUTHORITYITEM STANDARD="Evangelicalism">
        <AUTHORITYITEM STANDARD="Fascism"/>
        <AUTHORITYITEM STANDARD="Hinduism"/>
        <AUTHORITYITEM STANDARD="Jansenism"/>
        <AUTHORITYITEM STANDARD="Judaism"/>
        <AUTHORITYITEM STANDARD="Liberal Judaism"/>
        <AUTHORITYITEM STANDARD="Mormonism"/>
        <AUTHORITYITEM STANDARD="Sufism"/>
        <AUTHORITYITEM STANDARD="Utilitarianism"/>
        <AUTHORITYITEM STANDARD="Zen Buddhism"/>
        <AUTHORITYITEM STANDARD="Zionism"/>

**Comment by alliyya** [comment]
I kind of understand the intention of using these as orgnames as the `DENOMINATION`  tag was unavailable in the event or writing schema. Just speculating that this was a way that all 3 types of documents could point to the same religion in a way. Might be something to consider for revising these entries or in terms of extraction.

**Comment by SusanBrown** [comment]
Good point. I suppose for the time being we could just extract and point to them them as religions without making a final decision on what to do about them within the original tagging Orlando context. Is that the best route @lemaka @alliyya ?

**Comment by alliyya** [comment]
These are entries(13) that standard name does not match what's available in my authority file for orgs
A few of them have a empty standard name(2), this could be a result of my stale data files but will be checking what's online. 

For now going to check if there's reg attribute and see if that's in the authority file and point to that. (I think it was a miss tagging between the reg and standard for some of these) But there are still a few instances where this won't suffice as there is no reg.(5)

So able to map about 8 of the 13 odd ones. But in whole they probably should be revised. Especially the ones without standard names.

```xml
Beauvoir, Simone de
<orgname reg="Sorbonne,, University of Paris" standard="">Sorbonne</orgname>

Blackwell, Antoinette Brown
<orgname reg="Congregational Church" standard="Congregational">Congregational</orgname>

Equiano, Olaudah
<orgname reg="Anglican Church" standard="Church of England">Church of England</orgname>

Heaney, Seamus
<orgname standard="">St Columb's College</orgname>

Howe, Julia Ward
<orgname standard="New England Woman Suffrage Association">New England Woman Suffrage Association</orgname>

Hunt, Violet
<orgname orgtype="South Kensington Art School" reg="Royal College of Art" standard="South Kensington School of Art">South Kensington School of Art</orgname>

Hutchinson, Lucy
<orgname reg="Independent Church" standard="Independent"><denomination self-defined="SELFYES">Independent</denomination></orgname>

Jameson, Storm
<orgname standard="King's College,, London University"><school institutionlevel="POST-SECONDARY" reg="King's College,, London University" studentbody="CO-ED">King's College, University of London</school></orgname>

McWilliam, Candia
<orgname standard="St George's"><school institution="PRIVATE" institutionlevel="PRIMARY" reg="St George's High School for Girls,, Edinburgh">St George's</school></orgname>

Muir, Willa
<orgname reg="University of St Andrews" standard="St Andrews University">St Andrews University</orgname>

Ní Chuilleanáin, Eiléan
<orgname reg="Ursuline Convent School,, Cork" standard="Ursuline Convent School"><school institutionlevel="SECONDARY" reg="Ursuline Convent School,, Cork" religious="RELIGIOUSYES" studentbody="SINGLESEX">Ursuline Convent School</school></orgname>

Renault, Mary
<orgname reg="Radcliffe Infirmary" standard="Radcliffe Infirmary, Oxford">Radcliffe Infirmary, Oxford</orgname>

Sherwood, Mary Martha
<orgname orgtype="Abbey School,, Reading" standard="Abbey School">Abbey School</orgname>
```

**Comment by alliyya** [comment]
Howe, Julia Ward
`<orgname standard="New England Woman Suffrage Association">New England Woman Suffrage Association</orgname>`
unable to map to this org
`<AUTHORITYITEM STANDARD="New England Woman Suffrage Assocation"/>`
Due to the misspelling in the authority list
